src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import java.util.Arrays;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+    }
+
```

Comment:
If you want you can create your own user with the constructor. Your code becomes more object-oriented. 

```
    public User(String email, String password) {
        this.email = email;
        this.password = password;
    }
```

So in the method main you have to use methods setEmail() and setPassword()